File: front-end/src/app/service-models/app/v1/files-in-db/application-file-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ApplicationFileSM extends CoinManagementServiceModelBase<number> {
    fileName!: string;
    fileType!: string;
    fileDescription!: string;
    fileBytes!: string;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/FilesInDb/ApplicationFileSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.FilesInDb
{
    public class ApplicationFileSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public ApplicationFileSM()
        {
        }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public string FileDescription { get; set; }

        ///NOTE : ONLY USE THIS METHOD IF THE FILE ID CAN BE SAVED IN DB TABLES DIFFERENTLY
        public byte[] FileBytes { get; set; }
    }
}
